python/ray/tests/test_placement_group.py

```diff
@@ -30,9 +30,10 @@ def __init__(self):
         def v(self):
             return 10
 
-    # bundle is placement group reserved resources and can't be used in bundles
+    # kBundle_ResourceLabel is placement group reserved resources and
+    # can't be used in bundles
     with pytest.raises(Exception):
-        ray.util.placement_group(bundles=[{"bundle": 1}])
+        ray.util.placement_group(bundles=[{"__ray_reserved_bundle_resource__": 1}])
```

Comment:
I don't think we need to rename "bundle" anymore?

Reply:
I think it should not be named "bundle" because that word is so common and restricts the user unnecessarily

Reply:
ok, fair enough.